src/main/kotlin/org/koil/admin/Models.kt

```diff
@@ -7,3 +7,5 @@ sealed class AdminAccountUpdateResult {
     data class EmailAlreadyTaken(val account: Account) : AdminAccountUpdateResult()
     object CouldNotFindAccount : AdminAccountUpdateResult()
 }
+
+data class AccountEnriched(val account: Account, val organizationName: String)
```

Comment:
I think in this case I'd want to denormalize the Organizationname into the Invoice- since Invoice should maintain the company name at the time the invoice was generated (regardless of further changes to the name).

 At the moment- I decided not to denormalize for the Organization Name because the Account's Org name should always change with the root and this was the only location the name was actually necessary on the Account. If we start needing to put the Org name in many places, then I think we should look into denormalizing it into the Account table.

Reply:
Sounds good.